src/data/gleanSql.js

```diff
@@ -0,0 +1,58 @@
+export const getGleanQuery = (columnName, table) =>
+  `
```

Comment:
When copy/pasting this will result in an extra blank line at the top of the SQL code. Is this intentional?